src/GRANULAR/fix_wall_gran.cpp

```diff
@@ -118,6 +119,12 @@ FixWallGran::FixWallGran(LAMMPS *lmp, int narg, char **arg) :
       kt /= force->nktv2p;
     }
     iarg = 10;
+    
+    if (strcmp(arg[iarg],"limit_damping") == 0) {
+      limit_damping = 1;
```

Comment:
I don't see any place where limit_damping is initialized to a default value (of 0). Is this code "valgrind clean"?

Reply:
I'll also fix this. Checking with Valgrind, this fix has a few extra, unrelated issues with uninitialized values. I'll patch them all and submit an update later. 

In terms of fix wall/gran/region, no changes were necessary as it calculates forces using the methods defined in the parent wall/gran class.